aws/plugin.go

```diff
@@ -315,6 +315,7 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"aws_servicequotas_service_quota":                              tableAwsServiceQuotasServiceQuota(ctx),
 			"aws_servicequotas_service_quota_change_request":               tableAwsServiceQuotasServiceQuotaChangeRequest(ctx),
 			"aws_ses_email_identity":                                       tableAwsSESEmailIdentity(ctx),
+			"aws_ses_domain_identity":                                      tableAwsSESDomainIdentity(ctx),
```

Comment:
```suggestion
			"aws_ses_domain_identity":                                      tableAwsSESDomainIdentity(ctx),
			"aws_ses_email_identity":                                       tableAwsSESEmailIdentity(ctx),
```

Can you please sort in alphabetical order?